### PR TITLE
[BIP-395] Delegate AAVE/stkAAVE to TokenLogic

### PR DESCRIPTION
**Pull request by Xeonus**
- BIP-395A: enable gov voting rights
- Tenderly-sim: https://dashboard.tenderly.co/public/safe/safe-apps/simulator/a8e899b9-2340-4512-bb0c-d2db3a324ba9
- BIP-395B: grant governance proposal power
- Tenderly-sim: https://dashboard.tenderly.co/public/safe/safe-apps/simulator/fa0cb741-6222-45ec-ba97-06bc0bc459d9

### CODE REVIEW

**Comment by Xeonus** [comment]
@Tritium-VLK : carefully review this payload. I was only able to create a json with the raw hex data based on karpatkey's template and looking at the proxy contract. I took the raw tx data from Karpatkey from here: https://safe-transaction-mainnet.safe.global/api/v1/safes/0x0EFcCBb9E2C09Ea29551879bd9Da32362b32fc89/multisig-transactions/

File: BIPs/2023-W31/BIP-395A.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "txBuilderVersion": "1.16.1",
+    "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x9abeb5c10c633c53bf8f71bea4d1c4379856c2f144d66106b84c4c315a77ce51"
+  },
+  "transactions": [
+    {
+      "to": "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9",
+      "value": "0",
+      "data": "0xdc937e1c0000000000000000000000002cc1ade245020fc5aae66ad443e1f66e01c54df10000000000000000000000000000000000000000000000000000000000000000",
```
Comment (Tritium-VLK) [inline BIPs/2023-W31/BIP-395A.json:17]:
```
c.decode_input("0xdc937e1c0000000000000000000000002cc1ade245020fc5aae66ad443e1f66e01c54df10000000000000000000000000000000000000000000000000000000000000000")
("delegateByType(address,uint8)", ['0x2cc1ADE245020FC5AAE66Ad443e1F66e01c54Df1', 0])
```

Comment (Tritium-VLK) [reply BIPs/2023-W31/BIP-395A.json:17]: 0x2cc1ADE245020FC5AAE66Ad443e1F66e01c54Df1 not found in reversebook

File: BIPs/2023-W31/BIP-395B.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "txBuilderVersion": "1.16.1",
+    "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
+    "createdFromOwnerAddress": "",
+    "checksum": "0x9abeb5c10c633c53bf8f71bea4d1c4379856c2f144d66106b84c4c315a77ce51"
+  },
+  "transactions": [
+    {
+      "to": "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9",
+      "value": "0",
+      "data": "0xdc937e1c0000000000000000000000002cc1ade245020fc5aae66ad443e1f66e01c54df10000000000000000000000000000000000000000000000000000000000000001",
```
Comment (Tritium-VLK) [inline BIPs/2023-W31/BIP-395B.json:17]: ("delegateByType(address,uint8)", ['0x2cc1ADE245020FC5AAE66Ad443e1F66e01c54Df1', 1])

Comment (Tritium-VLK) [reply BIPs/2023-W31/BIP-395B.json:17]: same address

File: BIPs/2023-W31/BIP-395A.json (hunk 1 above)
Comment (Tritium-VLK) [reply BIPs/2023-W31/BIP-395A.json:17]: https://github.com/BalancerMaxis/bal_addresses/pull/96

**Review by Tritium-VLK — APPROVED** [review]

File: BIPs/2023-W31/BIP-395A.json (hunk 1 above)
Comment (Xeonus) [reply BIPs/2023-W31/BIP-395A.json:17]: Adding full json

File: BIPs/2023-W31/BIP-395B.json (hunk 2 above)
Comment (Xeonus) [reply BIPs/2023-W31/BIP-395B.json:17]: Adding full json

**Pull request merged**